test/langtools/tools/javac/reflect/quoted/TestCaptureQuotable.java

```diff
@@ -61,7 +63,9 @@ public void testCaptureRefAndIntConstant() {
         Quotable quotable = (Quotable & ToIntFunction<Number>)y -> y.intValue() + hello.length() + x;
         Quoted quoted = quotable.quoted();
         assertEquals(quoted.capturedValues().size(), 2);
-        assertEquals(((Var)quoted.capturedValues().values().iterator().next()).value(), x);
+        Iterator<Object> it = quoted.capturedValues().values().iterator();
```

Comment:
This test had a bug and was testing the wrong captured values. This is now fixed.